lib/Backends/OpenCL/OpenCL.cpp

```diff
@@ -399,14 +399,15 @@ void OpenCLFunction::executeConvolution(const OCLConvolutionInst *CC) {
   PaddingTLBR pads(CC->getPads());
   ShapeHW kdim(CC->getKernels());
   ShapeHW sdim(CC->getStrides());
+  unsigned group = CC->getGroup();
   // Create options for compiling the program.
   // Don't use names M, N, K as they are defined in precompiled headers.
 
   std::vector<std::string> options;
   // Number of spacial axes.
   addIntOption(options, "v_nax", 2);
   // Number of groups.
-  addIntOption(options, "v_g", 1);
+  addIntOption(options, "v_g", group);
```

Comment:
Add an assert that group is not > 1. And a comment that fast convolution kernels have issues with groups > 1 and that this assert should be removed once these issues are fixed.

Reply:
This assertion has been added at the front of this function.